recipe/build.sh

```diff
@@ -1,28 +1,21 @@
 #!/bin/env bash
 
+# FIXME: This is a hack to make sure the environment is activated.
+# The reason this is required is due to the conda-build issue
+# mentioned below.
+#
+# https://github.com/conda/conda-build/issues/910
+#
+source activate _build
```

Comment:
Can remove this after this issue ( https://github.com/conda/conda-build/issues/910 ) is fixed.